shared/pages/PartialClosure/PartialClosure.js

```diff
@@ -124,7 +146,7 @@ export default class PartialClosure extends Component {
       sellCurrency: getCurrency,
       sellAmount: getAmount,
       buyAmount: haveAmount,
-      destinationSellAddress: (customWalletUse && this.customWalletAllowed()) ? customWallet : null,
+      destinationSellAddress: (this.customWalletAllowed()) ? customWallet : null,
```

Comment:
Двойные скобки лишние
```suggestion
      destinationSellAddress: this.customWalletAllowed() ? customWallet : null,
```